src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -9,6 +9,14 @@
            (click)="enableRgwModule()">
            Enable RGW Module</a>
       </cd-alert-panel>
+      <cd-alert-panel   *ngIf="restartGatewayMessage"
+                        type="warning"
+                        spacingClass="mb-3"
+                        i18n>Please restart all Ceph Object Gateway instances in all zones to ensure consistent multisite configuration updates.
```

Comment:
```suggestion
                        i18n>Please restart all participating Ceph Object Gateway instances in default zone to ensure consistent while doing Import and Export
```

Reply:
Daemon restart is necessary after migrate to multisite or if a user created a new realm/zg/zone on fresh deployment..so decided to go on with a general message